### PR TITLE
ESRP signing for master and fix help check

### PR DESCRIPTION
**Pull request by markcowl**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
signing job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/15/

File: CodeSign.targets [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<UsingTask TaskName="ESRPSignTask" AssemblyFile="$(CISignRepoPath)\tools\sdkbuildtools\tasks\MS.Az.Sdk.OnPremise.Build.Tasks.dll" />
```
Comment (MiYanni) [inline CodeSign.targets]: Any reason this file is using tabs instead of spaces?

File: tools/RemoveCodeSignArtifacts.ps1 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+if ($Path -eq $null) {
+    $Path=$PSScriptRoot
+} else {
+    $Path = Resolve-Path $Path
+}
+
+Write-Output "Under the'$Path' folder"
+
+"Signed","Unsigned" | ForEach-Object {
```
Comment (MiYanni) [inline tools/RemoveCodeSignArtifacts.ps1]: If this script is basically just removing 2 folders, please do this instead in the build.proj:
```xml
<Exec Command="$(PowerShellCoreCommandPrefix) &quot;Remove-Item -Path $(SignedDirectory),$(UnsignedDirectory) -Recurse -Force -Confirm:$false -ErrorAction Ignore&quot;" ContinueOnError="WarnAndContinue" IgnoreExitCode="true" />
```

Then, we don't need to maintain yet another script.

**Comment by vladimir-shcherbakov** [comment]
test sign build : https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/18/console

**Comment by markcowl** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/20/

**Pull request merged**